lib/pbench/test/functional/server/test_datasets.py

```diff
@@ -483,12 +490,12 @@ def test_details(self, server_client: PbenchServerClient, login_user):
                 assert (
                     d.metadata["dataset.metalog.pbench"]["script"]
                     == detail["runMetadata"]["script"]
-                )
+                ), f"Unexpected details {detail}"
```

Comment:
Are you sure you want to provide a failure message for this assertion?  Without it, I believe PyTest will print the values of both `d.metadata["dataset.metalog.pbench"]["script"]` and `detail["runMetadata"]["script"]`; with it, I think it will only print your message (but, I could be wrong on that...).

Reply:
The idea was to get the entire `run-data` index document rather than just the (missing/incorrect) subset. (Although, now that it's not Saturday anymore, I guess it if was "missing" it would just throw an unhandled `KeyError` rather than failing the assert anyway, so this was likely pointless...)